R/opts.R

```diff
@@ -12,8 +12,13 @@
 #' @param max deprecated; use `dist` instead
 #' @param fixed deprecated; use `dist` instead
 #' @param prior_weight deprecated; prior weights are now specified as a
-#' model option. Use the `weigh_delay_priors` argument of
-#' [estimate_infections()] instead.
+#' model option. Use the `weight_prior` argument instead
+#' @param weight_prior Logical; if TRUE (default), the generation time prior
```

Comment:
Should this be generic since it's inherited by the other delays?

```suggestion
#' @param weight_prior Logical; if TRUE (default), the prior
```